glib/src/auto/functions.rs

```diff
@@ -861,3 +864,97 @@ pub fn uuid_string_is_valid(str: &str) -> bool {
 pub fn uuid_string_random() -> crate::GString {
     unsafe { from_glib_full(ffi::g_uuid_string_random()) }
 }
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_check_windows_version")]
+pub fn win32_check_windows_version(
+    major: i32,
+    minor: i32,
+    spver: i32,
+    os_type: Win32OSType,
+) -> bool {
+    unsafe {
+        from_glib(ffi::g_win32_check_windows_version(
+            major,
+            minor,
+            spver,
+            os_type.into_glib(),
+        ))
+    }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_error_message")]
+pub fn win32_error_message(error: i32) -> crate::GString {
+    unsafe { from_glib_full(ffi::g_win32_error_message(error)) }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_ftruncate")]
+pub fn win32_ftruncate(f: i32, size: u32) -> i32 {
```

Comment:
This is unsound. The `f` here is a file descriptor and if you pass random integers then bad things might happen

Reply:
Marked the function as ignored + a TODO comment. Don't have the time to deal with windows bits now